northd/northd.c

```diff
@@ -7202,7 +7208,12 @@ build_pre_acls(struct ovn_datapath *od,
      * which handles defragmentation, in order to match L4 headers. */
     if (od->has_stateful_acl) {
         for (size_t i = 0; i < od->n_router_ports; i++) {
-            skip_port_from_conntrack(od, od->router_ports[i],
+            struct ovn_port *op = od->router_ports[i];
+            if (op->peer && is_l3dgw_port(op->peer) &&
+                enable_conntrack(op->nbsp)) {
```

Comment:
Instead of querying the config map here I'd rather add a new field to `struct ovn_port`, e.g., `needs_conntrack` and populate that one when we build the port, probably somewhere in `join_logical_ports`.

Also, should we log when a user configures the new option on a different type of logical switch port?

Reply:
@dceara thanks for your suggestions！
i will upgrade a new patch